package.json

```
@@ -10,10 +10,12 @@
     "prettier-check": "prettier --check **/*.{js,json,scss,html}",
     "prettier-format": "prettier --write **/*.{js,json,scss,html}",
     "test": "yarn prettier-check && yarn workspace ssr test && yarn workspace client test",
-    "build": "cd stumptown && npm run build-json && cd .. && yarn workspace client build && yarn workspace ssr build && node cli/index.js ssr --build-html",
+    "build:content": "cd stumptown && npm run build-json && cd ..",
```

```suggestion
    "build:content": "cd stumptown && npm run build-json",
```
I actually tried and this works just as well. I.e. without the `cd ..` you can still run: `yarn build:content && yarn build:workspaces` fine.